actions/setup-yq/action.yml

```diff
@@ -0,0 +1,65 @@
+name: Install yq
+description: Sets up yq yaml parser - https://github.com/mikefarah/yq
+runs:
+  using: composite
+  steps:
+    - id: download-yq
+      shell: bash
+      run: |
+        set -e  # Exit on error
```

Comment:
These scripts should exit on error by default. See flags in the bash command at https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#defaultsrunshell